src/morphing.rs

```diff
@@ -1,8 +1,12 @@
+use rand::Rng;
 use rand::os::OsRng;
 
 use pad::*;
 use objects::*;
 use parsing::{parse_target_size, parse_objects};
+use distribution::{sample_object_count, sample_html_size, sample_object_sizes};
+
+const PAGE_SAMPLE_LIMIT: u8 = 10;
```

Comment:
From the Rust book:

"So how do you know which type of integer to use? If you’re unsure, Rust’s defaults are generally good choices, and integer types default to i32: it’s generally the fastest, even on 64-bit systems. The primary situation in which you’d use isize or usize is when indexing some sort of collection."

I'm generally in favor of defaulting to letting the compiler choose the data type for us unless there's a good reason not to.

Reply:
Forgot that for constants the type must be annotated.